### PR TITLE
disable command warnings yum/rpm

### PR DESCRIPTION
**Pull request by FLiPp3r90**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,12 +31,11 @@ You can customize your selection by editing ```defaults/main.yml```
 Installation
 ------------
 
-```$ ansible-galaxy install CSCfi.hp-firmware-upgrade ```
```
Comment (oscarkraemer) [inline README.md:31]: Why did you remove this?

File: example/example-playbook.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    upgrade_ilo_firmware: False
+    upgrade_system_rom_firmware: False
+    upgrade_disk_controller_firmware: False
+    upgrade_disk_drive_firmware: False
+    upgrade_power_management_controller_firmware: False
+    upgrade_intel_network_adapter_firmware: False
+    upgrade_mellanox_network_adapter_firmware: False
+    upgrade_qlogic_network_adapter_firmware: False
+  tasks:
+    - include_tasks: ../roles/hp-firmware-upgrade/tasks/main.yml
```
Comment (oscarkraemer) [inline example/example-playbook.yml:13]: This is making assumptions about that role name. For example if you have a requirement.yml file.

File: tasks/main.yml [hunk 3]
```diff
@@ -1,5 +1,9 @@
 ---
-- include_tasks: prerequisite.yml
+- include_tasks: ../roles/hp-firmware-upgrade/tasks/find_hp_gen.yml
+  tags:
+    - find_hp_gen
+
+- include_tasks: ../roles/hp-firmware-upgrade/tasks/prerequisite.yml
```
Comment (oscarkraemer) [inline tasks/main.yml:6]: I don't understand why we would want to change the path of the task? Maybe this is some practice that I'm not aware of.

**Review by oscarkraemer — CHANGES_REQUESTED** [review]
Hi,
Thanks for your PR. I have some questions about the changes of the role name and the changes how the include_tasks are used.

File: README.md (hunk 1 above)
Comment (FLiPp3r90) [reply README.md]: Hi, in my oppinion this is not necessary to explain how to install an Ansible role. If you do not agree this, I can insert this again.

File: example/example-playbook.yml (hunk 2 above)
Comment (FLiPp3r90) [reply example/example-playbook.yml]: Oh sorry, this was an internal change. We can leave the old role name as well.

File: tasks/main.yml (hunk 3 above)
Comment (FLiPp3r90) [reply tasks/main.yml:6]: The problem is, that this role use a main.yml as starting point. If you use a playbook to start the tasks in the role and you include the role, the sequence of the tasks in the role are alphabetical. So I insert the include task in the playbook so we can start with the main.yml. The problem is, that the include_task directive in the role uses the playbook directory, too. So we have to set the relative path in the role task to successfully execute each task in the role.

File: README.md (hunk 1 above)
Comment (FLiPp3r90) [reply README.md:31]: Hi, in my oppinion this is not necessary to explain how to install an Ansible role. If you do not agree this, I can insert this again.

File: example/example-playbook.yml (hunk 2 above)
Comment (FLiPp3r90) [reply example/example-playbook.yml:13]: Oh sorry, this was an internal change. We can leave the old role name as well.

File: tasks/main.yml (hunk 3 above)
Comment (martbhell) [reply tasks/main.yml:6]: @FLiPp3r90 would you be able to share a playbook where you are using this role in the manner you are describing? I don't follow. 

https://github.com/CSCfi/ansible-role-hp-firmware-upgrade/blob/master/tests/test.yml#L21

is how we use the role right now and everything happens in order it is written in the role, not in alphabetical.

Comment (FLiPp3r90) [reply tasks/main.yml:6]: Hi, thats realy strange. I will reproduce this behavior as soon as possible. 
Here is my playbook thats actually running:

```yaml
- name: Upgrade HP Firmware
  hosts: all
  vars:
    upgrade_ilo_firmware: False
    upgrade_system_rom_firmware: False
    upgrade_disk_controller_firmware: False
    upgrade_disk_drive_firmware: False
    upgrade_power_management_controller_firmware: False
    upgrade_intel_network_adapter_firmware: False
    upgrade_mellanox_network_adapter_firmware: False
    upgrade_qlogic_network_adapter_firmware: False
  tasks:
    - include_tasks: ../roles/hp-firmware-upgrade/tasks/main.yml

Comment (martbhell) [reply tasks/main.yml:6]: Thanks for checking. 
Always fun to hear about new uses of ansible! I also edited your message to make it look nice in github :)

**Comment by FLiPp3r90** [comment]
I need to debug some stuff. I will delete this pull request and open a new one if all bugs are resolved.
